specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2023-09-01/Clusters.json

```diff
@@ -0,0 +1,622 @@
+    },
+    "ClusterListResult": {
+      "type": "object",
+      "properties": {
+        "nextLink": {
+          "description": "The link used to get the next page of recommendations.",
+          "type": "string"
+        },
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Cluster"
+          },
+          "description": "A list of Log Analytics clusters."
+        }
+      },
+      "description": "The list clusters operation response."
+    },
+    "keyVaultProperties": {
+      "type": "object",
+      "description": "The key vault properties.",
+      "properties": {
+        "keyVaultUri": {
+          "description": "The Key Vault uri which holds they key associated with the Log Analytics cluster.",
+          "type": "string"
+        },
+        "keyName": {
+          "description": "The name of the key associated with the Log Analytics cluster.",
+          "type": "string"
+        },
+        "keyVersion": {
+          "description": "The version of the key associated with the Log Analytics cluster.",
+          "type": "string"
+        },
+        "keyRsaSize": {
+          "description": "Selected key minimum required size.",
+          "type": "integer",
+          "format": "int32"
+        }
+      }
+    },
+    "billingType": {
+      "type": "string",
+      "readOnly": false,
+      "description": "Configures whether billing will be only on the cluster or each workspace will be billed by its proportional use. This does not change the overall billing, only how it will be distributed. Default value is 'Cluster'",
+      "enum": [
+        "Cluster",
+        "Workspaces"
+      ],
+      "x-ms-enum": {
+        "name": "BillingType",
+        "modelAsString": true
+      }
+    },
+    "ClusterSku": {
+      "type": "object",
+      "description": "The cluster sku definition.",
+      "properties": {
+        "capacity": {
+          "description": "The capacity reservation level in Gigabytes for this cluster.",
+          "type": "integer",
+          "format": "int64",
+          "enum": [
+            100,
+            200,
+            300,
+            400,
+            500,
+            1000,
+            2000,
+            5000,
+            10000,
+            25000,
+            50000
+          ],
+          "x-ms-enum": {
+            "name": "Capacity"
+          }
+        },
+        "name": {
+          "type": "string",
+          "description": "The SKU (tier) of a cluster.",
+          "enum": [
+            "CapacityReservation"
+          ],
+          "x-ms-enum": {
+            "name": "ClusterSkuNameEnum",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "AssociatedWorkspace": {
+      "type": "object",
+      "description": "The list of Log Analytics workspaces associated with the cluster.",
+      "properties": {
+        "workspaceId": {
```

Comment:
Added uuid explicitly as format.

Reply:
Failing lint diff review, if you can approve it that will be helpful, otherwise its out of scope of current change from our perspective, we cannot postpone it currently due to features current public preview and release schedule.

Reply:
Pl add a suppression for this linter rule failure

[https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4183364&view=logs&j=0574a2a6-2d0a-5ec6-40e4-4c6e2f70bea2&t=80c3e782-49f0-5d1c-70dd-cbee57bdd0c7&l=43293](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4183364&view=logs&j=0574a2a6-2d0a-5ec6-40e4-4c6e2f70bea2&t=80c3e782-49f0-5d1c-70dd-cbee57bdd0c7&l=43293)